### PR TITLE
Components

### PR DESCRIPTION
**Pull request by samiksha-syn**

### CODE REVIEW

File: src/Components/SeasonCard.tsx/Index.tsx [hunk 1]
```diff
@@ -0,0 +1,351 @@
+import React from 'react'
```
Comment (mahipalsingh-syt) [inline src/Components/SeasonCard.tsx/Index.tsx:1]: remove extension from folder "SeasonCard.tsx"

File: src/Components/SeasonCard.tsx/Index.tsx [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+  cropIconName: 'settings',
+  cropImageStyle: {},
+  startDateTitle: 'Start',
+  endDateTitle: 'End',
+  editTitle: 'Edit',
+  deleteTitle: 'Delete',
+  syncStatus: 'none',
+  onEdit: () => Alert.alert('Warning', 'Add Edit function'),
+  startDate: new Date().getUTCDate() + '/' + new Date().getMonth() + '/' + new Date().getFullYear(),
+  endDate: new Date().getUTCDate() + '/' + new Date().getMonth() + '/' + new Date().getFullYear()
```
Comment (mahipalsingh-syt) [inline src/Components/SeasonCard.tsx/Index.tsx:154]: create a function for 

```new Date().getUTCDate() + '/' + new Date().getMonth() + '/' + new Date().getFullYear()```

File: src/Components/SeasonCard.tsx/Index.tsx (hunk 1 above)
Comment (samiksha-syn) [reply src/Components/SeasonCard.tsx/Index.tsx]: fixed

File: src/Components/SeasonCard.tsx/Index.tsx (hunk 2 above)
Comment (samiksha-syn) [reply src/Components/SeasonCard.tsx/Index.tsx]: fixed

**Pull request merged**